api/proto/config/appconfig.proto

```diff
@@ -86,4 +86,9 @@ message AppInstanceConfig {
   // container in the order they are listed, e.g., the first VM image
   // will be the root disk.
   repeated string volumeRefs = 14;
+
+  // the UUID of the NetworkAdapter which App Container stats collection
```

Comment:
The UUID is referring to the network instance to which the adapter is attached, right?

Reply:
Right. it's the same as 'networkId' inside the 'NetworkAdapter' API in netconfig.proto

Reply:
I'm think maybe we should change this to the 'staticIPAddress' which the UI/cloud has assigned for this adaptor instead of translating this into a NI UUID to simplify the logic on the EVE. It should not make much difference in the zedcloud, but much easier on the EVE.